package.json

```diff
@@ -379,7 +379,8 @@
     "swc-loader": "^0.2.6",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33279&commitId=58b5f6ca600852cc729176e8260175415fea8d6d&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|5| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|2825| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2213| Changed|
|Drawer.overlay drawer full.chromium.png|3320| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>